### ISSUE TITLE
wikiconfig.py documentation and example

### ISSUE THREAD

**Issue by UlrichB22** [moinwiki/moin#1605]
- The documentation in docs/admin/configure.rst needs some updates in the sections starting with the following lines:

https://github.com/moinwiki/moin/blob/85707b987088e1f3b9637f7d32472c5e98a6a9a8/docs/admin/configure.rst#L858

https://github.com/moinwiki/moin/blob/85707b987088e1f3b9637f7d32472c5e98a6a9a8/docs/admin/configure.rst#L1384

create_simple_mapping is not used in the current wikiconfig.py any more.

- There is an old example wikiconfig.py in docs/examples/config. Can it be removed to eliminate redundancy?

**Comment by RogerHaase** [comment]
yes

**Comment by ClaasTeichmann** [comment]
As a user, I found it confusing that there are several examples, that are quite similar. One example with the important settings would be nice, though, wherever it best resides.

**Comment by RogerHaase** [comment]
After I wrote that:

>A future simplification would include removing support for wikiconfig_local.py and wikiconfig_editme.py.

I tried to stop using wikiconfig_editme, but found it was very useful for testing various uncommon features and a time-saver, especially when wikiconfig.py was unstable. 

Per Claas's comment of too many similar examples, I am thinking that wikiconfig_editme should be stripped down to the first 7 lines of code (plus a `pass`) and the starting comment changed to say its primary purpose is for developers to test features not configured in the default wikiconfig.py.

With regard to docs/examples/config, I agree kill it. There is also the #1578 issue. Be aware there is "A real-life example of a wikiconfig.py can be found in the docs/examples/config/ directory." in configure.rst.
